### ISSUE TITLE
No bash syntax highlighting

### ISSUE THREAD

**Issue by evolutionxbox** [tuchk4/storybook-readme#120]
Using the following markdown:

```md
```js
import Button from 'components/button';
```​

```sh
#!/bin/bash

# example of using arguments to a script
echo "My first name is $1"
echo "My surname is $2"
echo "Total number of arguments is $#"
```​
```

Shows no syntax highlighting for the shell script.

![image](https://user-images.githubusercontent.com/660368/54072934-1f975c00-4279-11e9-9935-5c5e950ff0a0.png)

**Comment by evolutionxbox** [comment]
I don't know how to handle this.

I understand that only `jsx` and `typescript` syntaxes are being highlighted as they are the most popular.
I also understand that not many others will likely want or even use bash script highlighting.

Any ideas?

**Comment by tuchk4** [comment]
I hope this will be fixed in next release. 
I am goung to change highlite library.

**Comment by tuchk4** [comment]
released at `storybook-readme@5.0.0-alpha.150`. Now should highlight correctly.

Note that right now there are __breaking changes at alpha version__. I wrote about here https://github.com/tuchk4/storybook-readme/issues/118#issuecomment-472684902

**Comment by tuchk4** [comment]
fixed in `5.0.0`
